tests/phpunit/tests/block-supports/layout.php

```diff
@@ -173,4 +173,306 @@ function test_outer_container_not_restored_for_aligned_image_block_with_themejso
 
 		$this->assertSame( $expected, wp_restore_image_outer_container( $block_content, $block ) );
 	}
+
+	/**
+	 * Generates the CSS corresponding to the provided layout.
+	 *
+	 * @ticket 56467
+	 *
+	 * @dataProvider data_wp_get_layout_style
+	 *
+	 * @covers ::wp_get_layout_style
```

Comment:
`@covers` should be above `@dataProvider`. [See here](https://core.trac.wordpress.org/ticket/56070) for the order of annotations.